organizer_ui.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+"""
+Created on Fri May  1 20:59:21 2020
+
+@author: embden
+"""
+from kivy.config import Config
+Config.set('graphics', 'width', '480')
+Config.set('graphics', 'height', '640')
+
+from kivy.app import App
+from kivy.uix.button import Button
+from kivy.uix.label import Label
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.scrollview import ScrollView
+from kivy.core.window import Window
+
+# Regulate button height through this parameter.
+BUTTON_HEIGHT = 70
+
+class TaskButton(Button):
+    """
+    Class representing a button for a task expanding standard kivy button.
+
+    The distinctive feature of the class is its capability to be in two states:
+    active or waiting. In active state a button shold look like a push button.
+    Thus, when a user starts to work on a task he pushes the button, and when
+    finishes - releases it.
+    """
+    
+    def __init__(self, **kwargs):
+        """ Expand Button kivy class. """
+        super(TaskButton, self).__init__(**kwargs)
```

Comment:
```suggestion
       super().__init__(**kwargs)
```
просто сахарок для 3 питона)